worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -261,6 +261,37 @@ function isLastAction(action) {
     return ACTIONS[ACTIONS.length - 1 ].name === action;
 }
 
+function onProcessDocument(e) {
+    processWorkersVersions(e.rootDocument);
```

Comment:
I feel like we should add these fields to subdocuments as well?  Did we discuss this?